projects/plutus/test/test_current_sense.c

```diff
@@ -40,27 +44,27 @@ void setup_test(void) {
   s_callback_runs = 0;
   s_test_input_voltage = 0;
 
-  s_adc_settings = (LtcAdcSettings){ .mosi = { GPIO_PORT_B, 15 },
-                                     .miso = { GPIO_PORT_B, 14 },
-                                     .sclk = { GPIO_PORT_B, 13 },
-                                     .cs = { GPIO_PORT_B, 12 },
-                                     .spi_port = SPI_PORT_2,
-                                     .spi_baudrate = 750000,
-                                     .filter_mode = LTC_ADC_FILTER_50HZ_60HZ };
+  LtcAdcSettings adc_settings = (LtcAdcSettings){ .mosi = { GPIO_PORT_B, 15 },
```

Comment:
You don't need `(LtcAdcSettings)`. Please fix your formatting. Make sure you have trailing commas.

Reply:
Done